Formula/pplacer.rb

```diff
@@ -0,0 +1,34 @@
+class Pplacer < Formula
+  # Matsen_2010: "https://dx.doi.org/10.1186/1471-2105-11-538"
+  desc "Place query sequences on a fixed reference phylogenetic tree"
+  homepage "https://matsen.fhcrc.org/pplacer/"
+
+  # We use binaries to avoid compiling OCaml code
+  if OS.mac?
+    url "https://github.com/matsen/pplacer/releases/download/v1.1.alpha17/pplacer-Darwin-v1.1.alpha17.zip"
```

Comment:
This software has been `v1.1alpha*` since 2010! 😂

Reply:
Yeah, I know. They developed it a bit ahead of the time, and Erick thought `pplacer` would likely get superseded by other implementations in `RAxML` etc. But so many (new) pipelines still use it.

Reply:
Want to open an issue and ask the upstream authors to tag a stable 1.2.0 release? And perhaps to read https://semver.org. =)

Reply:
Not particularly... I don't think he/they are actively working on it anymore. [epa-ng](https://github.com/brewsci/homebrew-bio/blob/master/Formula/epa-ng.rb) is somewhat of a follow-up for `pplacer`, and [gappa](https://github.com/brewsci/homebrew-bio/blob/master/Formula/gappa.rb) for `guppy`.

Reply:
But some many other pipelines depend on it, there's no real way to avoid it.